libs/sdk/src/errors/mcp.error.ts

```
@@ -0,0 +1,281 @@
+
+  /**
+   * Get the internal error message (for logging)
+   */
+  getInternalMessage(): string {
+    return this.message;
+  }
+
+  /**
+   * Convert to MCP error response format
+   */
+  toMcpError(isDevelopment: boolean = false): {
```

ESLint

[error] Type boolean trivially inferred from a boolean literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)